internal/web/app/swagger.json

```diff
@@ -0,0 +1,644 @@
+{
```

Comment:
does this file need to be included in the commit? or will it be updated at runtime? it's been a while since I played with swagger

Reply:
It is served as a static file by the web server. So it has to be there. The Dockerfile builds it every time we build the image.